packages/components/src/tools-panel/stories/typography-panel.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalFontAppearanceControl as FontAppearanceControl,
+	__experimentalFontFamilyControl as FontFamilyControl,
+	__experimentalLetterSpacingControl as LetterSpacingControl,
+	LineHeightControl,
+} from '@wordpress/block-editor';
+import {
+	FontSizePicker,
+	__experimentalRadio as Radio,
+	__experimentalRadioGroup as RadioGroup,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { ToolsPanel, ToolsPanelItem } from '..';
+import Panel from '../../panel';
+import { ControlLabel } from '../../ui/control-label';
+import { FormGroup } from '../../ui/form-group';
+
+// These options match the theme.json typography schema
+const fontFamilies = [
+	{
+		fontFamily:
+			'-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif',
+		slug: 'system-fonts',
+		name: 'System Fonts',
+	},
+	{
+		fontFamily:
+			'Consolas, "Andale Mono WT", "Andale Mono", "Lucida Console", "Lucida Sans Typewriter", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Liberation Mono", "Nimbus Mono L", Monaco, "Courier New", Courier, monospace',
+		name: 'Monospace',
+		slug: 'monospace',
+	},
+	{
+		fontFamily:
+			'Hoefler Text, Baskerville Old Face, Garamond, Times New Roman, serif',
+		name: 'Serif',
+		slug: 'hoefler-times-new-roman',
+	},
+];
+const fontSizes = [
+	{
+		slug: 'small',
+		size: '1rem',
+		name: 'Small',
+	},
+	{
+		slug: 'normal',
+		size: '1.25rem',
+		name: 'Normal',
+	},
+	{
+		slug: 'medium',
+		size: '1.5rem',
+		name: 'medium',
+	},
+	{
+		slug: 'large',
+		size: '2rem',
+		name: 'Large',
+	},
+	{
+		slug: 'extra-large',
+		size: '3rem',
+		name: 'Extra large',
+	},
+];
+
+const LetterCaseControl = ( props ) => {
```

Comment:
I guess for this PR we should use the existing control [TextTransformControl](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/text-transform-control/index.js) ,no?

Reply:
I went ahead and mocked up our own version for this one, because the behavior is simple and the `RadioGroup` component is already closer to what we want both in terms of look and semantics.

I believe this part of the new FontSizePicker will also be an instance of `RadioGroup`:

<img src="https://user-images.githubusercontent.com/555336/135953852-a18f1390-f48b-44a7-aa2b-443eabb60ae4.png" alt="Preset font size picker" width="300">

So one of our subtasks would be to update the styling of `RadioGroup`.

Reply:
Have you already had a look at [`ToggleGroupControl`](https://wordpress.github.io/gutenberg/?path=/story/components-togglegroupcontrol--default)?

Reply:
Ah, there we go! 😆 Will replace.

Reply:
> Replaced and filed related issues #35381 #35382 #35383

Thank you so much!